### PR TITLE
Intuitive list

### PR DESCRIPTION
**Pull request by emieldesmidt**
This PR will make the UI Significantly more intuitive, and informative.
Utilises cardview components:

image

### CODE REVIEW

**Comment by PraveshMoelchand** [comment]
I'll add the stripped date to the Alertdialog as well

File: voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt [hunk 1]
```diff
@@ -91,17 +88,15 @@ class VotingActivity : AppCompatActivity() {
...
      */
     private fun showNewVoteDialog() {
-        val builder: AlertDialog.Builder = AlertDialog.Builder(this)
-        builder.setTitle("Create proposal")
 
-        val input = EditText(this)
-        input.inputType = InputType.TYPE_CLASS_TEXT
-        input.hint = "p != np"
-        builder.setView(input)
+        val mDialogView = LayoutInflater.from(this).inflate(R.layout.initiate_dialog, null)
```
Comment (PraveshMoelchand) [inline voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt:92]: Why the `m` in `mDialogView`

File: voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt [hunk 2]
```diff
@@ -91,17 +88,15 @@ class VotingActivity : AppCompatActivity() {
...
+
+        val mBuilder = AlertDialog.Builder(this)
```
Comment (PraveshMoelchand) [inline voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt:94]: Same here, why the `m`

**Comment by PraveshMoelchand** [comment]
Also, the kotlin-ipv8 submodule should be on commit `6b76301bb74bc3d2421368329f4e6d8ddfc78985`

File: voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt (hunk 1 above)
Comment (PraveshMoelchand) [reply voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt:92]: Had an offline discussion with @emieldesmidt, this is apparently not need, so renaming now

File: voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt (hunk 2 above)
Comment (PraveshMoelchand) [reply voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt:94]: See https://github.com/emieldesmidt/trustchain-superapp/pull/14#discussion_r410174633

**Comment by erwinvanthiel** [comment]
Proposing a new vote leads to an exception, message says: proposalInput must not be null. Input was not null however. Probably a small bug but needs to be fixed before merge

**Comment by emieldesmidt** [comment]
should be fixed now

**Comment by PraveshMoelchand** [comment]
As can be seen in de voting result, it works now
afbeelding

**Pull request merged**